ptbcontrib/log_forwarder/log_forwarder.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+import logging
+from collections.abc import Iterable
+from telegram.constants import ParseMode
+from telegram.ext import ExtBot
+
+
+class LogForwarder(logging.Handler):
+    def __init__(
+        self,
+        bot: ExtBot,
+        chat_ids: Iterable[int],
+        parse_mode: ParseMode = ParseMode.MARKDOWN_V2,
+        log_levels: list[str] = ["WARN", "ERROR"],
```

Comment:
mutable default values are discouraged. that's also what codacy complains about here. I recommend either setting the default to a tuple or `None` and then converting `None` to the default list in `__init__`

Reply:
I removed the default mutable value and put in a tuple instead and changed the type to `Iterable[str]`, however codacy keeps complaining about complexity of code increasing.